src/PlatformCheck.cpp

```diff
@@ -0,0 +1,19 @@
+// Copyright 2005-2018 The Mumble Developers. All rights reserved.
+// Use of this source code is governed by a BSD-style license
+// that can be found in the LICENSE file at the root of the
+// Mumble source tree or at <https://www.mumble.info/LICENSE>.
+
+#include "murmur_pch.h"
+
+#include <QtCore/QLibrary>
+
+#include "PlatformCheck.h"
+
+bool PlatformCheck::IsWine() {
+#ifdef Q_OS_WIN
+	if (QLibrary::resolve(QLatin1String("ntdll.dll"), "wine_get_version") != NULL) {
```

Comment:
Maybe we should point to https://wiki.winehq.org/Developer_FAQ#How_can_I_detect_Wine.3F ?

*Added [again] because other comment was on outdated code*